sentry-android-core/src/main/java/io/sentry/android/core/SpanFrameMetricsCollector.java

```diff
@@ -277,7 +279,7 @@ private static int interpolateFrameCount(
     final long frameMetricsDurationNanos = frameMetrics.getTotalDurationNanos();
     final long nonRenderedDuration = spanDurationNanos - frameMetricsDurationNanos;
     if (nonRenderedDuration > 0) {
-      return (int) (nonRenderedDuration / frameDurationNanos);
+      return (int) Math.ceil((double) nonRenderedDuration / frameDurationNanos);
```

Comment:
not sure if `ceil` here now clashes a little with [addPendingFrameDelay](https://github.com/getsentry/sentry-java/blob/fd1151b5c7696d17711c40c2facee553a43d07c3/sentry-android-core/src/main/java/io/sentry/android/core/SpanFrameMetricsCollector.java#L285), I assume it has the same goal (to round the remaining time in a finished span to include one more frame)?

Reply:
It shouldn't clash, because `addPendingFrameDelay` only adds a frame if it's slow.
Also, it considers the last frame duration, which is subtracted in this function (`frameMetrics.getTotalDurationNanos`)

Reply:
mm, but if `addPendingFrameDelay` adds a slow frame, we will also add it here don't we?
Like let's say the span is 50ms, and we have one normal frame and 34ms left which were all consumed by a slow frame. `addPendingFrameDelay` would add 1 frame, but then we come here and add 3 more? Or am I missing something obvious?

Reply:
yep